hieradata/common.yaml

```
@@ -247,7 +247,7 @@ profile::buildmaster::default_tools:
     sourceURL: https://github.com/adoptium/temurin17-binaries/releases/download
   maven:
     mvn:
-      version: "3.8.4"
+      version: "3.8.5"
```

The 3.8.6 version [hasn't been released yet](https://github.com/apache/maven/releases) but the corresponding tag exists: https://github.com/apache/maven/tags
```suggestion
      version: "3.8.6"
```